src/medal_app/views.cljs

```diff
@@ -18,26 +16,31 @@
    [:td (+ gold silver bronze)]])
 
 (defn medal-body []
-  (let [loading (re-frame/subscribe [::subs/loading])
+  (let [loading? (re-frame/subscribe [::subs/loading?])
         medals (re-frame/subscribe [::subs/medals])
-        sortOrder (re-frame/subscribe [::subs/sortOrder])]
+        sort-order (re-frame/subscribe [::subs/sort-order])]
     [:div
-      (when @loading "Loading...")
-      [:div {:class "container"}
-        [:h2 "MEDAL COUNT"]
-        [:table  
+      (when @loading? "Loading...")
+        [:div {:class "container"}
+         [:h2 "MEDAL COUNT"]
+         [:table  
           [:thead  
-            [:tr  
-              [:th]
-              [:th {:on-click #(re-frame/dispatch [::events/sort-medals "gold" @medals]) :class (if (= @sortOrder "gold") "active" "")}
-                [:div.legend-gold]]
-              [:th {:on-click #(re-frame/dispatch [::events/sort-medals "silver" @medals]) :class (if (= @sortOrder "silver") "active" "")}
-                [:div.legend-silver]]
-              [:th {:on-click #(re-frame/dispatch [::events/sort-medals "bronze" @medals]) :class (if (= @sortOrder "bronze") "active" "")}
-                [:div.legend-bronze]]
-              [:th {:on-click #(re-frame/dispatch [::events/sort-medals "total" @medals]) :class (if (= @sortOrder "total") "active" "")} "Total"]]]
-          [:tbody  
-      (map-indexed medal-list @medals)]]]]))
+           [:tr  
+            [:th]
+            [:th {:on-click #(re-frame/dispatch [::events/sort-medals "gold" @medals])
+                  :class (if (= @sort-order "gold") "active" "")}
+             [:div.legend-gold]]
+            [:th {:on-click #(re-frame/dispatch [::events/sort-medals "silver" @medals])
+                  :class (if (= @sort-order "silver") "active" "")}
+             [:div.legend-silver]]
+            [:th {:on-click #(re-frame/dispatch [::events/sort-medals "bronze" @medals])
+                  :class (if (= @sort-order "bronze") "active" "")}
+             [:div.legend-bronze]]
+            [:th {:on-click #(re-frame/dispatch [::events/sort-medals "total" @medals])
+                  :class (if (= @sort-order "total") "active" "")}
+              "Total"]]]
+          [:tbody
+           (map-indexed medal-row @medals)]]]]))
```

Comment:
Changed indentation to single-space indents for Reagent elements, to match our RCP conventions. For other Clojure functions we keep two-space indents.